### PR TITLE
chore: update pr template

### PR DESCRIPTION
**Pull request by MDunitz**
### Reviewers
**Functional:** 

**Readability:** 

---

## Changes
- add
- remove
- modify

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,13 +5,14 @@
...
 ---
 
+
 ## Changes
 - add
 - remove
 - modify
 
-## Definition of Done (from ticket)
+- #TICKET_NUMBER
```
Comment (atolopko-czi) [inline PULL_REQUEST_TEMPLATE.md:14]: 👏   What about putting this first?

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -5,13 +5,14 @@
...
 
 ## QA steps (optional)
 
-## Known Issues
+## Known Issues or Open Questions
```
Comment (atolopko-czi) [inline PULL_REQUEST_TEMPLATE.md:18]: I usually place comments directed at the reviewer in the PR itself, and rarely use this. But if others use it, I'd definitely keep it.  I suppose this is similar to a section we used on Meta team called "Notes for Reviewer" (or something similar).

**Review by atolopko-czi — APPROVED** [review]
Thanks for the improvements!

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (metakuni) [reply PULL_REQUEST_TEMPLATE.md:18]: One question... how will known issues/open questions noted in a PR be tracked or addressed in the future?

Comment (MDunitz) [reply PULL_REQUEST_TEMPLATE.md:18]: I like the section for more meta comments/questions. Like a question about the validity of an overall approach. I like "Notes for Reviewer" I'll swap that in.

**Pull request merged**